tmt/schemas/plan.yaml

```diff
@@ -0,0 +1,199 @@
+  description:
+    $ref: "/schemas/core#/definitions/description"
+
+  # https://tmt.readthedocs.io/en/stable/spec/plans.html#spec-plans-discover
+  discover:
+    oneOf:
+      - $ref: "/schemas/discover/fmf#"
+      - $ref: "/schemas/discover/shell#"
+      - type: array
+        items:
+          anyOf:
+            - $ref: "/schemas/discover/fmf#"
+            - $ref: "/schemas/discover/shell#"
+
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#enabled
+  enabled:
+    $ref: "/schemas/core#/definitions/enabled"
+
+  # https://tmt.readthedocs.io/en/stable/spec/plans.html#environment
+  environment:
+    $ref: "/schemas/common#/definitions/environment"
+
+  # https://tmt.readthedocs.io/en/stable/spec/plans.html#environment-file
+  environment-file:
+    $ref: "/schemas/common#/definitions/array_of_strings"
+
+  # https://tmt.readthedocs.io/en/stable/spec/plans.html#execute
+  execute:
+    oneOf:
+      - $ref: "/schemas/execute/tmt#"
+      - $ref: "/schemas/execute/upgrade#"
+      - type: array
+        items:
+          $ref: "/schemas/execute/tmt#"
+      - type: array
+        items:
+          $ref: "/schemas/execute/upgrade#"
+
+  # https://tmt.readthedocs.io/en/stable/spec/plans.html#finish
+  finish:
+    oneOf:
+      - $ref: "/schemas/finish/ansible#"
+      - $ref: "/schemas/finish/shell#"
+      - type: array
+        items:
+          anyOf:
+            - $ref: "/schemas/prepare/ansible#"
+            - $ref: "/schemas/prepare/shell#"
+
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#link
+  link:
+    $ref: "/schemas/core#/definitions/link"
+
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#order
+  order:
+    $ref: "/schemas/core#/definitions/order"
+
+  # https://tmt.readthedocs.io/en/stable/spec/plans.html#prepare
+  prepare:
+    oneOf:
+      - $ref: "/schemas/prepare/ansible#"
+      - $ref: "/schemas/prepare/install#"
+      - $ref: "/schemas/prepare/shell#"
+      - type: array
+        items:
+          anyOf:
+            - $ref: "/schemas/prepare/ansible#"
+            - $ref: "/schemas/prepare/install#"
+            - $ref: "/schemas/prepare/shell#"
+
+  # https://tmt.readthedocs.io/en/stable/spec/plans.html#provision
+  provision:
+    oneOf:
+      - $ref: "/schemas/provision/artemis#"
+      - $ref: "/schemas/provision/connect#"
+      - $ref: "/schemas/provision/container#"
+      - $ref: "/schemas/provision/local#"
+      - $ref: "/schemas/provision/virtual#"
+      - type: array
+        items:
+          anyOf:
+            - $ref: "/schemas/provision/artemis#"
+            - $ref: "/schemas/provision/connect#"
+            - $ref: "/schemas/provision/container#"
+            - $ref: "/schemas/provision/local#"
+            - $ref: "/schemas/provision/virtual#"
+
+  # https://tmt.readthedocs.io/en/stable/spec/plans.html#provision
+  report:
+    oneOf:
+      - $ref: "/schemas/report/display#"
+      - $ref: "/schemas/report/html#"
```

Comment:
We're missing the `junit` report plugin.

Reply:
added junit plugin